src/biokbase/narrative/tests/narrative_mock/mockclients.py

```diff
@@ -291,9 +291,7 @@ def check_job(self, params):
         job_id = params.get("job_id")
         if not job_id:
             return {}
-        job_state = self.job_state_data.get(
-            job_id, {"job_id": job_id, "status": "unmocked"}
-        )
+        job_state = self.job_state_data.get(job_id, {"job_id": job_id})
```

Comment:
a default is supplied for "status", so we don't need the "unmocked" status value any more